utils/oscap-xccdf.c

```diff
@@ -605,35 +605,33 @@ int app_evaluate_xccdf(const struct oscap_action *action)
 		xccdf_session_remediate(session);
 	}
 
+	/* Get the result from TestResult model and decide if end with error or with correct return code */
+	int evaluation_result = xccdf_session_contains_fail_result(session) ? OSCAP_FAIL : OSCAP_OK;
+
+	/* syslog message */
+#if defined(HAVE_SYSLOG_H)
+	syslog(priority, "Evaluation finished. Return code: %d, Base score %f.", result,
```

Comment:
`result` should be also changed to `evaluation_result` because before the change the syslog call used to use result of the `xccdf_session_contains_fail_result(session) ? OSCAP_FAIL : OSCAP_OK` statement.